mailbox/api/src/main/java/org/apache/james/mailbox/MailboxListener.java

```diff
@@ -57,6 +62,56 @@
      */
     void event(Event event);
 
+    class QuotaEvent implements Event, Serializable {
```

Comment:
I don't think every quota events are about the value of quotas for a given quotaroot.
I would make another level in the hierarchy and name this one QuotaRootUsageUpdateEvent